advisories/a101963.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Technical Advisory 101963
+advisory: A-101963
+summary: In specific circumstances, a RESTORE operation may restore incorrect data from a backup that contains incorrect metadata related to revision history.
+toc: true
+affected_versions: v22.2.0 to v22.2.8 and v22.1.0 to 22.1.19
+advisory_date: 2023-05-09
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In the following two circumstances, backups that include [revision history](../stable/take-backups-with-revision-history-and-restore-from-a-point-in-time.html) may record incorrect metadata. The presence of this incorrect metadata could cause a subsequent `RESTORE` operation to write incorrect data:
```

Comment:
```suggestion
In the following two circumstances on CockroachDB v22.2.0 to v22.2.8 and v22.1.0 to 22.1.19, backups that include [revision history](../{{site.versions["stable"]}}/take-backups-with-revision-history-and-restore-from-a-point-in-time.html) may record incorrect metadata. The presence of this incorrect metadata could cause a subsequent [`RESTORE`](../{{site.versions["stable"]}}/restore.html) operation to write incorrect data:
```

Reply:
This isn't accurate, the first circumstance could affect either version but the second circumstance can only affect 22.2.